### ISSUE TITLE
_transform.default.extend is not a function under 2.14 beta

### ISSUE THREAD

**Issue by mwpastore** [emberjs/data#4979]
Custom transforms under 2.14 beta raise `TypeError: _transform.default.extend is not a function`. Seems like it might be a transpilation issue. 

I'm having trouble reproducing the error in Ember Twiddle because ED beta won't load with these errors:

* `Error while parsing the 'sandbox' attribute: 'allow-modals' is an invalid sandbox flag.`
* `` Error: Could not find module `ember-data/-private` imported from `ember-data/setup-container` ``

[Here](https://ember-twiddle.com/d61aafbfce82bec7c2cc33e564d00313)'s what I have so far. 

Falling back to 2.13.x resolves the issue.

**Comment by rwjblue** [comment]
Where are you importing the transform from?  Can you share your custom transform code?

**Comment by mwpastore** [comment]
Hi @rwjblue! Here's the code:

```javascript
import Transform from 'ember-data/transform';

import format from 'accounting/format-money';
import unformat from 'accounting/unformat';

export default Transform.extend({
  deserialize: str => str && unformat(str),
  serialize: str => str && format(str)
});
```

**Comment by mwpastore** [comment]
Changing:

```javascript
import Transform from 'ember-data/transform';
```

To:

```javascript
import Transform from 'ember-data/transforms/transform';
```

Seems to resolve the issue under 2.14 beta (but it is not backwards-compatible).

**Comment by stefanpenner** [comment]
On a side note, we should deprecate transforms being ember objects.  Just make them classes.

**Comment by rwjblue** [comment]
Looks like https://github.com/emberjs/data/blob/master/addon/transform.js just needs an export default?

**Comment by stefanpenner** [comment]
@rwjblue ah, we both found the same likely issue: https://github.com/emberjs/data/commit/24f41209b7ab9978bcdf029e50e3e62564d0b50f#commitcomment-22121947

**Comment by stefanpenner** [comment]
We should fix, and add a test to ensure those are importable.

**Comment by stefanpenner** [comment]
@runspired can you address?

**Comment by runspired** [comment]
@stefanpenner I can

**Comment by mwpastore** [comment]
This appears to be fixed in 2.14-release. Thanks @runspired!
